### PR TITLE
Revert "[Deploy] Allow multiple services to be restarted on master commit"

### PR DESCRIPTION
**Pull request by fleupold**
Reverts gnosis/dex-services#373

No longer needed, due to the fact that we can encode two target services in a single URL.

https://dfusion.auto.gnosisdev.com/services/dfusion-mainnet-master,dfusion-rinkeby-master/restart

### CODE REVIEW

File: docker/deploy.sh [hunk 1]
```diff
@@ -29,10 +18,11 @@ docker push $REGISTRY_URI:$image_name
...
-    for url in "${AUTODEPLOY_URL_LIST[@]}"; do
-      autodeploy "$url"
-    done
+if [ "$image_name" == "master" ] && [ -n "$AUTODEPLOY_URL" ] && [ -n "$AUTODEPLOY_TOKEN" ]; then
+    # Notifying webhook
+    curl -s --output /dev/null --write-out "%{http_code}" \
+    -H "Content-Type: application/json" \
+    -X POST \
+    -d '{"token": "'$AUTODEPLOY_TOKEN'", "push_data": {"tag": "'$AUTODEPLOY_TAG'" }}' \
+    $AUTODEPLOY_URL
```
Comment (nlordell) [inline docker/deploy.sh]: Can we fix the indentation here or would you prefer to do that as a separate PR?

Comment (fleupold) [reply docker/deploy.sh]: I can fix this separately given that this is clean revert
